server/config/service.go

```diff
@@ -90,8 +90,7 @@ func (s *service) newConfig() Config {
 	}
 }
 
-func (s *service) newInitializedConfig(newStoredConfig StoredConfig) (*Config, utils.Logger, error) {
-	log := s.NewBaseLogger()
+func (s *service) newInitializedConfig(newStoredConfig StoredConfig, log utils.Logger) (*Config, error) {
```

Comment:
Before this change, the logger created here would be returned and used in `OnActivate()` causing a lot of messages to not be channel-logged as this logger doesn't have the stored config.